rootfs/dev_requirements.txt

```diff
@@ -2,7 +2,4 @@
 coverage>=4.0
 
 # Run "make flake8" to check python syntax and style
-flake8==2.4.1
-
-# Used for mocking endpoints
-mock==1.0.1
+flake8==2.5.1
```

Comment:
is the version bump here necessary? If not would you mind cutting this into a separate commit just for separation's sake?